ynr/apps/elections/models.py

```diff
@@ -78,7 +78,7 @@ class Election(EEModifiedMixin, models.Model):
     UnsafeToDelete = Exception
 
     def __str__(self):
-        return self.name
+        return f"{self.name} {self.election_date}"
```

Comment:
This was added to improve the appearance of the election in the filtering drop down - however if this is likely to cause issues elsewhere, it can be reverted and there is a way to alter the image on the `QueuedImageFilter` instead

Reply:
What kind of issues? I think it's a very rare case where we need the date (when there is two elections in an area), but it does happen (see Castle Point). Another option would be to add another date filter to be populated once an election is selected.

Reply:
So there was one failing test as a result of the change, related to an error message - as it now expected a date to be included. But thinking about this again with a clear head, I think I will revert this change because I think its very hard to know for sure what the knock on effect is, and if it will cause some of our pages to look a bit unusual etc.